cloudinary-core/src/main/java/com/cloudinary/transformation/RadiusOption.java

```diff
@@ -0,0 +1,29 @@
+package com.cloudinary.transformation;
+
+import com.cloudinary.utils.StringUtils;
+
+public class RadiusOption {
```

Comment:
This class can be a private (or at most package-private) method in `Transformation`.